exporting/read_config.c

```diff
@@ -397,7 +397,13 @@ struct engine *read_exporting_config()
                                  CONFIG_BOOLEAN_YES:  CONFIG_BOOLEAN_NO))
             tmp_instance->config.options |= EXPORTING_OPTION_SEND_AUTOMATIC_LABELS;
         else
-            tmp_instance->config.options &= ~EXPORTING_OPTION_SEND_AUTOMATIC_LABELS;
+            if ((tmp_ci_list->exporting_type != EXPORTING_CONNECTOR_TYPE_GRAPHITE &&
+                  tmp_ci_list->exporting_type != EXPORTING_CONNECTOR_TYPE_GRAPHITE_HTTP))
+                tmp_instance->config.options &= ~EXPORTING_OPTION_SEND_AUTOMATIC_LABELS;
+            else
+                tmp_instance->config.options |= EXPORTING_OPTION_SEND_AUTOMATIC_LABELS;
+
+        netdata_log_error("KILLME %u %u", tmp_instance->config.options & EXPORTING_OPTION_SEND_AUTOMATIC_LABELS, tmp_ci_list->exporting_type);
```

Comment:
really? 😄

Reply:
:man_facepalming:
As you can see, I was testing. :sweat_smile: